src/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3'
+services:
+
+  db:
+    build:
+      context: ./db/build
+      args:
+        - sapw=Th3PA55--8zz
+    image: leaf_db
+    container_name: leaf_db_demo
+    ports:
+      - 1433:1433
+    networks:
+      - leaf
+
+  server:
+    build: ./server
+    image: leaf_api
+    container_name: leaf_api_demo
+    ports:
+      - 5001:5001
+    volumes:
+      - ./server/keys:/.keys
+      - ./server:/app
+      - server-log:/logs
+    environment:
+      - LEAF_APP_DB=db,1433;Database=LeafDB;uid=sa;Password=Th3PA55--8zz
```

Comment:
It looks like something in parsing this env var is splitting on `=`... You might want to run the container with bash to confirm the env vars.